Include/pymem.h

```diff
@@ -60,20 +60,21 @@ PyAPI_FUNC(void) PyMem_Free(void *ptr);
  * overflow checking is always done.
  */
 
-#define PyMem_New(type, n) \
-  ( ((size_t)(n) > PY_SSIZE_T_MAX / sizeof(type)) ? NULL :      \
-        ( (type *) PyMem_Malloc((n) * sizeof(type)) ) )
+#define PyMem_New(TYPE, N)                          \
+    (((size_t)(N) <= PY_SSIZE_T_MAX / sizeof(TYPE)) \
+        ? (TYPE *)PyMem_Malloc((N) * sizeof(TYPE))  \
+        : NULL)
 
 /*
- * The value of (p) is always clobbered by this macro regardless of success.
- * The caller MUST check if (p) is NULL afterwards and deal with the memory
- * error if so.  This means the original value of (p) MUST be saved for the
+ * The value of (PTR) is always clobbered by this macro regardless of success.
+ * The caller MUST check if (PTR) is NULL afterwards and deal with the memory
+ * error if so.  This means the original value of (PTR) MUST be saved for the
  * caller's memory error handler to not lose track of it.
  */
-#define PyMem_Resize(p, type, n) \
-  ( (p) = ((size_t)(n) > PY_SSIZE_T_MAX / sizeof(type)) ? NULL :        \
-        (type *) PyMem_Realloc((p), (n) * sizeof(type)) )
-
+#define PyMem_Resize(PTR, TYPE, N)                          \
```

Comment:
```PyMem_Resize(p, type, n)```
Would it be more appropriate to use lowercase, since `PyMem_Del` uses lowercase?

Reply:
PEP-7 recommends having uppercase parameter names. I decided to cleanup the code because the file is quite small.

Reply:
Oh， yes, I almost forgot :(